libxss.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: libxss
+  version: 1.2.4
+  epoch: 0
+  description: "Xlib-based X11 Screen Saver extension client library"
+  copyright:
+    - license: custom:XFREE86
```

Comment:
[licence](https://gitlab.freedesktop.org/xorg/lib/libxfont/-/blob/master/COPYING).

The same as [libx11](https://github.com/wolfi-dev/os/blob/main/libx11.yaml#L7)